minio/minio
# Added check for mandatory MINIO_KMS_KES_KEY_NAME env var

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
If MinIO started with KMS enabled, MINIO_KMS_KES_KEY_NAME should be set for server to start.

## Motivation and Context


## How to test this PR?
1. Setup a KES instance
2. Start MinIO server with with KES environment variables
```
MINIO_KMS_KES_ENDPOINT=https://127.0.0.1:7373 MINIO_KMS_KES_API_KEY=kes:v1:AE2apN/xEunVvpkFylHAfx4PQne+EbSEsaaLnvQzLpEe MINIO_KMS_KES_KEY_NAME=minio-default-key MINIO_KMS_KES_ENCLAVE=minio MINIO_KMS_KES_CAPATH=<path-to-kes-server.crt> minio server ./test
```

If `MINIO_KMS_KES_KEY_NAME` not set it should error out as below
```
ERROR The mandatory environment variable "MINIO_KMS_KES_KEY_NAME" not set: Invalid KES configuration
```

## Types of changes

## Checklist:
